package.json

```diff
@@ -7,7 +7,9 @@
   },
   "devDependencies": {
     "pulp": "^10.0.4",
-    "purescript-psa": "^0.5.0-rc.1",
+    "pulp": "^11.0.x",
+    "purescript": "0.11.x",
+    "purescript-psa": "^0.5.0",
```

Comment:
For the core libraries we prefer to fetch by hand - it means that we can update libraries before the `npm` release is done.